pandas/tests/frame/test_alter_axes.py

```diff
@@ -1056,6 +1056,38 @@ def test_reindex_signature(self):
                               "limit", "copy", "level", "method",
                               "fill_value", "tolerance"}
 
+    def test_droplevel(self):
+        # GH20342
+        df = pd.DataFrame([
+            [1, 2, 3, 4],
```

Comment:
can you add tests for series as well

Reply:
I did, on lines 1085-1089 :)

Reply:
oh not in THIS test. pls separate out and put in tests/series/test_alter_axes